### DISCUSSION TITLE
M5Core2 + LoRa433 v1.1 - Trying to install without success

### DISCUSSION THREAD

**Discussion by GonsoH2o** [dl9rdz/rdz_ttgo_sonde#475] [TTGO firmware]
Hello
I am trying to implement the firmware on my M5core2 AWS with a LoRa433 v1.1 but I am not succeeding....
Is there any place where I can find documentation or can you help me here?

many, many thanks in advance.

**Comment by dl9rdz** [comment]
What did you try, and what is not working?
1) Flashing the firmware
2) Does anything show in the display?
3) Does WiFi work (without any configuration there should show up an AP named RDZsonde)?

**Comment by GonsoH2o** [comment]
Hi, thanks for your quick answer!!
I tried:
 1. Flashing the firmware:  dev20240905b-C3-full.bin // devel20240521-B17-full.bin
 2. The display does not show anything
 3. Wifi don't work (any AP showed)

Flash download tool v3.9.7_2 --> Chip type ESP32 / Develop /(UART)
SPI SPEED -> 80MHHz / SPI MODE -> DIO / BAUD: 921600

regards

**Reply by dl9rdz** [reply]
Alright, then the best next step is to look at the serial log. 
- Easiest is to use Chrome or Edge browser and the web flash tool, there you can open the console.
- Alternative is to use any other serial terminal, like platform io (pio device monitor on the command line, or some GUI integration), or Arduino GUI

**Comment by GonsoH2o** [comment]
iT WORKS!! 

I've flashed the firmware thru the web flash tool, and it seems that starts to work. AP: RDZsonde is active and I can access to it to see the config options. 
Screen is showing a scan search for some frequencies and sondes.

Many thanks for all your help! Is ther any tip or change I must apply to the M5core2 to tune it correctly?

**Reply by dl9rdz** [reply]
Not much to fine tune, I have not done much testing on the M5 recently, maybe the screen layout is not ideal (but likely I will make an update later this year). Don't expect the SD card to work on the M5.

As to why it did not work with Flash download tool before, maybe you did not set the base address (0x1000 instead of 0). The web flash tool does use the right value automatically.

**Comment by GonsoH2o** [comment]
Many  thanks for your support. Much appreciated!.
I will investigate and I will try to do my bit, if possible.

regards
